defaults/main.yml

```diff
@@ -3,7 +3,7 @@
 
 openio_repository_products:
   sds:
-    release: '17.04'
+    release: '18.04'
```

Comment:
every 6 months, this default will change so?

Reply:
Do you want a `current` symlink to the latest and put this in the default value ?